config/routes.rb

```diff
@@ -41,6 +41,7 @@
       post '/inactivate', to: 'memberships#inactivate'
       post '/reactivate', to: 'memberships#reactivate'
       post '/accept', to: 'memberships#accept'
+      post '/withdraw', to: 'memberships#withdraw'
```

Comment:
This routes file is a bit messy 😅

```suggestion
      post :withdraw, on: :member
```